poly/src/polynomial/univariate/sparse.rs

```diff
@@ -62,11 +62,30 @@ impl<F: Field> Polynomial<F> for SparsePolynomial<F> {
         if self.is_zero() {
             return F::zero();
         }
+
+        let total_bits = core::mem::size_of::<usize>() * 8;
+        let max_pow_2 = total_bits - self.degree().leading_zeros() as usize;
+
+        let mut pows_2 = Vec::<F>::with_capacity(max_pow_2);
```

Comment:
This is computing `log2(total_bits)`, right? You can replace it with `let max_pow_2 = ark_std::log2(total_bits);`.

Reply:
fair enough

Reply:
imo this should be a usize default function

Reply:
btw, I am uncomfortable with the current name, imo it ought to be named `log2_then_ceil`, or `log2_of_next_pow_of_2` or something.